contracts/BaseStrategy.sol

```diff
@@ -16,6 +16,8 @@ struct StrategyParams {
     uint256 totalDebt;
     uint256 totalGain;
     uint256 totalLoss;
+    uint256 lossRatioLimit;
```

Comment:
im torn about this since this breaks compatibility at the ABI and vault/ strat level, but not sure how to make it work other than adding a different method or struct for this or doing the change strictly at the BaseStrategy level only

Reply:
WDYT about adding healthcheck params to a different struct separated from the strategy params?

Reply:
Something that I have noticed, adding changes at the end of a structure won't break the existing code as long as these changes arent't needed of course.

I have made a PR just to run the tests without the changes to BaseStrategy to demonstrate that.

https://github.com/yearn/yearn-vaults/actions/runs/877678032

Reply:
Yeah, actually this is true becuase of how ABI decoding works. It expects a particular size not a particular shape for a struct, and decodes it as basically a tuple.